.github/workflows/doxygen.yml

```diff
@@ -0,0 +1,30 @@
+name: Doxygen
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  Build and Publish Documentation:
+    name: Build and Publish Documentation
+    runs-on: ubuntu-latest
+    container:
+      image: novelrt/novelrt-build:latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Generate Documentation
+        run: scripts/cibuild.sh --documentation -DDOXYGEN_OUTPUT_DIR=$GITHUB_WORKSPACE/novelrt/docs
+
+      - name: Commit Documentation Updates
+        if: ${{ success() }}
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
```

Comment:
Is there docs on this e-mail or is there some "official-ness" to it?

Reply:
For this, I followed the example provided from the Checkout v2 Actions Repo (from GH) - https://github.com/actions/checkout

Reply:
This has been replaced - an e-mail was found as per https://github.community/t/github-actions-bot-email-address/17204/3 that will actually utilize the github-actions bot account for commits, PRs, etc.